File: src/main/java/com/matamercer/microblog/web/api/v1/dto/responses/LikeResponseDto.java
package com.matamercer.microblog.web.api.v1.dto.responses;

import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class LikeResponseDto extends BaseModelResponseDto{
    private BlogResponseDto liker;
}

File: src/main/java/com/matamercer/microblog/web/api/v1/dto/responses/LikeResponseDto.kt
package com.matamercer.microblog.web.api.v1.dto.responses

import lombok.Data
import lombok.EqualsAndHashCode

@EqualsAndHashCode(callSuper = true)
@Data
class LikeResponseDto : BaseModelResponseDto() {
    var liker: BlogResponseDto? = null
}